setup.cfg

```diff
@@ -3,3 +3,7 @@ universal = 1
 
 [metadata]
 long_description = file: README.rst
+
+[tool:pytest]
+filterwarnings =
+  ignore::marshmallow.warnings.ChangedInMarshmallow3Warning
```

Comment:
I'd keep these for our benefit. We do need to do this update sooner rather than later...

Reply:
Fair enough. The particular chage re: `strict` mode will be hard to miss when you end up supporting marshmallow 3, so I figured this was safe to ignore.

I could add a regex to match on the specific warning message. Something like

```
filterwarnings =
  ignore:strict=False:marshmallow.warnings.ChangedInMarshmallow3Warning
```

Reply:
That could work. My concern is more subtle behavior changes. Blowing up on changes to the signature of dump/load will hopefully be obvious enough to avoid causing problems.

Reply:
OK, I made the above change.